### ISSUE TITLE
YOLOX-S

### ISSUE THREAD

**Issue by wenguduo** [open-mmlab/mmdetection#8037]
When I change the loss_cls from cross_entropy_loss to FocalLoss in yolox_s_8x8_300e_coco.py ，the error show the follow error: 

File "/opt/conda/lib/python3.7/site-packages/mmcv/ops/focal_loss.py", line 37, in forward
    assert isinstance(target, (torch.LongTensor, torch.cuda.LongTensor))
AssertionError

How can I solve it, I have tried to change target type to LongTensor before train,but still error.

**Comment by hhaAndroid** [comment]
@wenguduo Sorry. We have not considered this situation for the time being. We will fix this as soon as possible.

**Comment by hhaAndroid** [comment]
@wenguduo You can use the following configuration instead, the effect is the same as focal loss.

```python
bbox_head=dict(
        type='YOLOXHead', num_classes=80, in_channels=128, feat_channels=128,
        loss_cls=dict(
            type='VarifocalLoss',
            use_sigmoid=True,
            reduction='sum',
            gamma=2.0,
            alpha=0.75,
            loss_weight=1.0)
    ),

```

**Comment by wenguduo** [comment]
Thanks a lot, I can run it

**Comment by wywywy01** [comment]
How VarifocalLoss  map?
